### PR TITLE
Node v2 `StoreChunks` and `GetChunks` endpoints

### PR DESCRIPTION
**Pull request by ian-shim**
## Why are these changes needed?
Implementation of v2 `StoreChunks` and `GetChunks` endpoints

## Checks

- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, in that case, please comment that they are not relevant.
- [ ] I've checked the new test coverage and the coverage percentage didn't drop.
- Testing Strategy
   - [ ] Unit tests
   - [ ] Integration tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: node/node_v2.go [hunk 1]
```diff
@@ -134,3 +131,26 @@ func (n *Node) DownloadBundles(ctx context.Context, batch *corev2.Batch) ([]*cor
 
 	return blobShards, rawBundles, nil
 }
+
+func (n *Node) ValidateBatchV2(
```
Comment (mooselumph) [inline node/node_v2.go:135]: Should this just go in the core/validator object?

**Review by mooselumph — APPROVED** [review]
Looks good. Idempotency as follow-up?

File: node/node_v2.go (hunk 1 above)
Comment (ian-shim) [reply node/node_v2.go:134]: I moved batch header validation as another method in validator object, but kept them separated

**Comment by ian-shim** [comment]
> Looks good. Idempotency as follow-up?

For now, I added [a check](https://github.com/Layr-Labs/eigenda/pull/893/files#diff-e0709c067d80ae092e5eb445765c32358b1161585f60e22f065f7e01e9fa90a3R59-R61) if the batch header has been stored and abort the entire store operation if it has been stored. 
We can add more sophisticated logic later but this would be on par with the v1 logic.

**Pull request merged**